src/Poseidon/Generator/Types.hs

```diff
@@ -1,38 +1,38 @@
-module Poseidon.Generator.Types (
-    IndWithAdmixtureSet (..),
-    AdmixtureSet (..),
-    PopulationWithFraction (..),
-    IndWithPosition (..),
-    SpatialTemporalPosition (..),
-    GenoEntry (..)
-) where
-
-import           Data.List                  (intercalate)
+module Poseidon.Generator.Types where
+
+import           Data.List      (intercalate)
 import           Poseidon.Janno
-import           SequenceFormats.Eigenstrat (GenoEntry (..))
 
-data IndWithAdmixtureSet = IndWithAdmixtureSet {
-      _admixInd  :: String
-    , _admixUnit :: String
-    , _admixSet  :: AdmixtureSet
+data IndAdmixpops = IndAdmixpops {
+      _indName   :: String
+    , _groupName :: String
+    , _popSet    :: [PopAdmixpops]
+} deriving (Show)
+
+data PopAdmixpops = PopAdmixpops {
+      _popName :: String
+    , _popFrac :: Rational
```

Comment:
Warum nimmst Du hier `Rational` und nicht einfach `Double`?

Reply:
I feels more correct, because a fraction is precisely what I want to express :thinking: